src/org/intellij/erlang/runconfig/ErlangRunConfigurationBase.java

```diff
@@ -86,7 +113,8 @@ public void readExternal(Element element) throws InvalidDataException {
 
   @Nullable
   @Override
-  public final RunningState getState(@NotNull Executor executor, @NotNull ExecutionEnvironment environment) throws ExecutionException {
+  public final RunningState getState(@NotNull Executor executor,
+                                     @NotNull ExecutionEnvironment environment) throws ExecutionException {
```

Comment:
Please avoid spurious formatting changes in your pull requests. It complicates browsing your changes.